my-react-app/src/Components/List/List.js

```diff
@@ -181,37 +183,54 @@ const List = ({ note, setNote }) => {
         </div>
       </Modal>
       <TableContainer component={Paper}>
-        <Table
-          className={classes.table}
-          size="small"
-          aria-label="a dense table"
-        >
+        <Table>
           <TableHead>
-            <TableRow>
-              <TableCell align="right">Имя</TableCell>
-              <TableCell align="right">Врач</TableCell>
-              <TableCell align="right">Дата(g)</TableCell>
-              <TableCell align="right">Жалоба</TableCell>
-              <TableCell align="right"></TableCell>
+            <TableRow className="table-head">
+              <TableCell className="table-20 table-grey" align="center">
+                Имя
+              </TableCell>
+              <TableCell className="table-20 table-grey" align="center">
+                Врач
+              </TableCell>
+              <TableCell className="table-10 table-grey" align="center">
+                Дата(g)
+              </TableCell>
+              <TableCell className="table-30 table-grey" align="center">
+                Жалоба
+              </TableCell>
+              <TableCell
+                className="table-10 table-grey"
+                align="center"
+              ></TableCell>
             </TableRow>
           </TableHead>
-          <TableBody>
+          <TableBody className="table-body">
             {note?.map((list, index) => (
               <TableRow key={`list-${index}`}>
-                <TableCell align="right">{list.patient}</TableCell>
-                <TableCell align="right">{list.doctor}</TableCell>
-                <TableCell align="right">{list.date}</TableCell>
-                <TableCell align="right">{list.vine}</TableCell>
-                <TableCell align="right">
+                <TableCell className="table-20 table-border" align="center">
+                  {list.patient}
+                </TableCell>
+                <TableCell className="table-20 table-border" align="center">
+                  {list.doctor}
+                </TableCell>
+                <TableCell className="table-10 table-border" align="center">
+                  {list.date}
+                </TableCell>
+                <TableCell className="table-40 table-border" align="center">
```

Comment:
why is the dimension of the columns with names different from the columns with data?